_episodes/43.md

```diff
@@ -0,0 +1,200 @@
+run daily migrations, you would not benefit from caching. However, one of the
+most common use cases where Trino shines is interactive adhoc analytics. This 
+type of querying is very fast in Trino, especially when using modern storage 
+formats like Iceberg.
+
+### Two types of caching
+
+There are two types of caching used with Trino. The first type caches the
+results of a common query or sub query to optimize performance for any query
+that overlaps with predicates to obtain the cached results. 
+
+The other type of query is object file caching. Rather than store the results of
+the query, you are caching the files from a file or object store that are
+scanned as part of the query.
+
+In this episode, we will focus on the latter type of caching. This will apply to
+connectors like Hive, Iceberg, Delta Lake, and Hudi.
+
+### Hive connector caching
+
+Trino has an [embedded caching engine](https://trino.io/docs/current/connector/hive-caching.html)
+in the Hive connector. This is convenient as it ships with Trino, however, it 
+does not work outside the Hive connector. The caching engine is 
+[Rubix](https://github.com/qubole/rubix) which has not been developed over two
+years. While this system works for simple Hive use cases, it fails to address
+use cases outside of Hive and hasn't been maintained since 2020. There are many
+features missing like kubernetes and containerization, security features, 
+support for more compute engines. 
+
+### What is Alluxio?
+
+Alluxio is world’s first open source data orchestration technology for analytics
+and AI for the cloud. It provides a common interface enabling computation
+frameworks to connect to numerous storage systems through a common interface.
+Alluxio’s memory-first tiered architecture enables data access at speeds orders
+of magnitude faster than existing solutions. Alluxio was originally developed at
+Berkley Amp Labs, [formerly known as Tachyon](
+https://amplab.cs.berkeley.edu/wp-content/uploads/2014/11/2014_socc_tachyon.pdf),
+and was less focused on caching and data orchestration and more focused on
+fault-tolerance via lineage and other techniques borrowed from Spark. 
+
+Alluxio lies between data driven applications, such as Trino and Apache Spark,
+and various persistent storage systems, such as Amazon S3, Google Cloud Storage,
+HDFS, Ceph, and Minio. Alluxio unifies the data stored in these different
+ storage systems, presenting unified client APIs and a global namespace to its
+upper layer data driven applications.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/43/alluxio-architecture.png"/>
+</p>
+
+Alluxio is commonly used as a distributed shared caching service so compute
+engines talking to Alluxio can transparently cache frequently accessed data,
+especially from remote locations, to provide in-memory I/O throughput. Alluxio
+also makes using Object storage much simpler as you will not hit the storage
```

Comment:
object

Reply:
also .. this sentence feels wrong ... it doesnt really make things simpler does it?

Reply:
I'll avoid using "simpler" but rather specify that "they enable unifying all data sources to a single file namespace". This makes things simpler in some aspects if you have multiple object storages across different storage technologies, object storage in different regions, or stored across different clouds.